pinot-query-runtime/src/test/resources/queries/OrderBy.json

```diff
@@ -0,0 +1,221 @@
+{
+  "basic_order_by": {
+    "tables": {
+      "basic": {
+        "schema": [
+          {"name": "col0", "type": "INT"},
+          {"name": "col1", "type": "INT"},
+          {"name": "col2", "type": "STRING"}
+        ],
+        "inputs": [
+          [1, 2, "a"],
+          [2, 3, "b"],
+          [3, 1, "c"],
+          [4, 4, "d"],
+          [5, 5, "e"],
+          [6, 6, "f"]
+        ]
+      }
+    },
+    "queries": [
+      {"sql": "SELECT * FROM {basic} ORDER BY col1 LIMIT 2 OFFSET 1"},
```

Comment:
Can we add a test for offest 0, offset 100, limit 100 and Limit all (psql supports this). as well as limit null statement

Reply:
I changed the `OFFSET 10` and `LIMIT 10` clauses to be 100 (they were meant to change the same thing) and added `OFFSET 0` and `LIMIT ALL`.

Reply:
Sorry, didn't do the math. yeah. the 10 is doing the same thing for 100.